doc/task-hierarchy.md

```diff
@@ -0,0 +1,160 @@
+Task Type
+- Abuse Detection
+- Answer Generation: Commonsense Question Answering
+- Answer Generation: Contextual Question Answering
+- Answer Generation: Fill in the Blank
+- Answer Generation: Multiple Choice Question Answering
+- Answer Generation: Open Question Answering
+- Author Identification
+- Classification
+- Combinatorics
+- Contextual Question Generation
+   - Generate questions based on given context e.g. a task to create a question based on a paragraph.
+- Coreference: Entity Coreference
+- Coreference: Pronoun Disambiguation
+- Counting
+   - Count an attribute of input e.g. a task to count number of vowels in a given word
+- Dialogue Understanding
+- Document Understanding
+- Entity Detection
+- Ethical Judgement
+- Explanation Generation
+- Fake News Detection
+- Grammar Error Correction
+- Hallucination
+   - Given a context, generate imaginary content e.g. given a sentence, generate a story/poem.
+- Handling Commands
```

Comment:
What does "Handling" mean? Is handling == executing?

Reply:
Yes, changing it to executing.